schema/manifest_backwards_compatibility_test.go

```diff
@@ -49,10 +49,9 @@ func TestBackwardsCompatibilityManifestList(t *testing.T) {
 		fail         bool
 	}{
 		{
-			digest: "sha256:219f4b61132fe9d09b0ec5c15517be2ca712e4744b0e0cc3be71295b35b2a467",
+			digest: "sha256:ce939cd86094ec89abba1168d55257dfe74781f438916737d5c0905d7afb8803",
 			manifestlist: `{
    "schemaVersion": 2,
-   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
```

Comment:
I don't think we want these, because (as we've [covered before][1]) `manifest_backwards_compatibility_test.go` is about “are we compatible with caught-in-the-wild blobs blobs?” and making local adjustments like this (even to a field which we will be ignoring in the OCI framework) defeats that purpose.

[1]: https://github.com/opencontainers/image-spec/pull/510#discussion_r94909550